make/dev.mk

```diff
@@ -0,0 +1,106 @@
+ifndef DEV_MK
+DEV_MK:=# Prevent repeated "-include".
+
+include ./make/verbose.mk
+include ./make/git.mk
+
+DOCKER_REPO?=quay.io/openshiftio
+IMAGE_NAME?=tektoncd-pipeline-operator
+REGISTRY_URI=quay.io
+
+DEVCONSOLE_OPERATOR_IMAGE?=quay.io/redhat-developers/tektoncd-pipeline-operator
```

Comment:
do we need a DEVCONSOLE here?

Reply:
Sorry I will update that to reflect our operator